cloudinit/util.py

```diff
@@ -2080,18 +2080,22 @@ def boottime():
     import ctypes
     import ctypes.util
 
-    NULL_BYTES = b"\x00"
-
     class timeval(ctypes.Structure):
         _fields_ = [("tv_sec", ctypes.c_int64), ("tv_usec", ctypes.c_int64)]
 
     libc = ctypes.CDLL(ctypes.util.find_library("c"))
     size = ctypes.c_size_t()
     size.value = ctypes.sizeof(timeval)
+    mib_values = [  # This corresponds to
```

Comment:
We can assume these two values are equal across *BSDs and stable as they belong to the kernel API, right?

Assuming that, it is not worth loading `CTL_KERN` and `KERN_BOOTTIME` symbols from `sysctl`, because of code complexity and performance impact. Or what's your take on this?